watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Utilities/Utility.java

```diff
@@ -118,4 +120,13 @@ public static void showAndBuildDialog(Activity activity, String title, String me
         builder.show();
     }
 
+    public static String[] COLORS = { "#F44336", "#673AB7", "#03A9F4", "#4CAF50", "#FF5722", "#607D8B",
+                                      "#00BCD4", "#9C27B0", "#E91E63", "#3F51B5", "#009688" };
+    public static String[] SECONDARY_COLORS = { "#D32F2F", "#512DA8", "#0288D1", "#388E3C", "#E64A19", "#455A64",
```

Comment:
unless we need it without a context, in which case we wouldn't be able to access resources